sdk/cognitivelanguage/azure-ai-language-questionanswering/azure/ai/language/questionanswering/aio/_operations/_patch.py

```diff
@@ -26,13 +27,14 @@
 class QuestionAnsweringClientOperationsMixin(QuestionAnsweringClientOperationsMixinGenerated):
```

Comment:
Same overloads for `get_asnwers` and `get_answers_from_text` should be present for async operations as well